exe/Main.hs

```diff
@@ -134,11 +134,12 @@ main = do
         let grab file = fromMaybe (head sessions) $ do
                 cradle <- Map.lookup file filesToCradles
                 Map.lookup cradle cradlesToSessions
+        let loadGhcSession =
```

Comment:
Why do we setup the environment slightly differently for the CLI mode and the LSP mode? Can we not use `loadGhcSessionIO =<< getCurrentDirectory` here?

Reply:
The difference was already there. The CLI mode used the `grab` function for creating cradles/sessions and my change doesn't change that aspect. Do you want the `grab` function to be deleted? I'm not sure how it affects the behavior in the CLI mode.

Reply:
I think I know what you meant. Now that `optGhcSession` is back in `IdeOptions`, `loadGhcSession` is gone.